docs/modules/vertiq_23xx_family.rst

```diff
@@ -0,0 +1,145 @@
+
+.. _vertiq_23xx_family:
+
+*********************************
+Vertiq 23-XX Family 
+*********************************
+
+.. image:: ../_static/module_pictures/23xx_family.png
+        :alt: Vertiq 23-XX Family
+        :width: 500
+        :align: center
+
+.. csv-table:: Vertiq 23-XX Family of Modules
+        :header: "Size", "Kv", "Default Firmware", "Available Firmware"
+        :align: center
+
+        "23-06", "220", "Servo", "Servo"
+        "23-06", "2200", "Speed", "Speed, Pulsing"
+        "23-14", "920", "Speed", "Servo, Speed"
+        
+Hardware Setup Walkthrough
+===============================
+
+Changing or Updating Firmware
+===============================
+
+Please follow the instructions found in :ref:`updating_firmware`
+
+Getting Started
+===============================
+
+Complete the correct "Getting Started Guide" for your module's style (if you are using a servo module, please complete "Getting Started with Vertiq Servo Modules")
+
+        * :ref:`Getting Started with Vertiq Speed Modules <control_center_tutorial>`
+        * :ref:`Getting Started with Vertiq Servo Modules <manual_angle_control_mechanisms>`
+
+.. I am putting these just to have some sort of placeholder link. Eventually we'll have actual getting started manuals
+
+More Features
+===============================
+
+Once you have completed the proper "Getting Started Guide," you can begin to dive deeper into your module's capabilities. Below, you will find
+a summary of all features supported on your module, IQUART Clients it can reach, as well as applicable tutorials. Please ensure that you are reading the feature
+summary for your module's style.
+
+Speed Module - Supported Features
```

Comment:
One little thing, might it be nicer to have Speed Module, Servo Module, and Pulsing Module as headers under Features here, and then have the supported features, clients, and tutorials as sub-headers under each of those? May just be a bit easier to navigate